client/src/public/ts/sliding-text.ts

```
@@ -1,44 +1,58 @@
     text: string,
-    textStyle: PIXI.TextStyle,
+    textStyle: TextStyle,
     slideBezier: BezierCurve,
     fadeBezier: BezierCurve,
-    duration: number
+    duration: number,
+    enemyID: string
   ) {
+    // use the sliding text if it already exists
+    if (Object.keys(SlidingText.slidingTexts).includes(enemyID)) {
+      return SlidingText.slidingTexts[enemyID];
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)